src/dynamic_batch_scheduler.cc

```diff
@@ -54,7 +54,8 @@ FinishSkippedRequests(
 {
   for (auto& queue : requests) {
     for (auto& request : queue) {
-      InferenceRequest::RespondIfError(request, response_status, true);
+      InferenceRequest::RespondIfError(
```

Comment:
It looks like `FinishSkippedRequests` is called for both `rejected` and `cancelled` requests. So this might do some double counting, and may not label them correctly as it currently is.

Can you elaborate on what this function is doing differently, and why it's needed if we call `RespondIfError` in the rejected/cancelled cases already? CC @kthui who may be able to help explain as well.

Reply:
NVM, it turned out one usage was in dynamic batcher and the other was in sequence batcher.